scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/ForkerUtils.java

```diff
@@ -48,6 +52,29 @@
     /** System property Key of the fork method */
     public static final String FORK_METHOD_KEY = "pas.launcher.forkas.method";
 
+    // allows to configure the runasme fork method for a workflow task
+    public static final String RUNAS_METHOD_GENERIC_INFO = "RUNAS_METHOD";
+
+    // allows to configure the runasme user for a workflow task
+    public static final String RUNAS_USER_GENERIC_INFO = "RUNAS_USER";
+
+    // allows to configure the runasme user domain for a workflow task
+    public static final String RUNAS_DOMAIN_GENERIC_INFO = "RUNAS_DOMAIN";
+
+    // allows to configure the runasme password for a workflow task (if runasme method is PWD)
+    public static final String RUNAS_PWD_GENERIC_INFO = "RUNAS_PWD";
```

Comment:
Does it mean that the password value can be given directly in the GI without using the third party credentials?

Reply:
yes. In that case it should reference a hidden variable. It's possible but not thoroughly recommended